tests/test_client.py

```diff
@@ -1,7 +1,101 @@
-# from unittest.mock import patch
-#
-# from dacsspace.client import ArchivesSpaceClient
+import os
+import shutil
+import unittest
+from unittest.mock import Mock, patch
 
+from asnake.client.web_client import ASnakeAuthError
 
-def test_client():
-    pass
+from dacsspace.client import ArchivesSpaceClient, ASnakeConfigError
+
+
+class ArchivesSpaceClientTests(unittest.TestCase):
+    def setUp(self):
+        """Move existing config file and replace with sample config."""
+        if os.path.isfile("dacsspace/local_settings.cfg"):
+            shutil.move("dacsspace/local_settings.cfg", "dacsspace/local_settings.old")
+        shutil.copy("dacsspace/local_settings.example", "dacsspace/local_settings.cfg")
+
+    @patch("requests.get")
+    @patch("requests.post")
+    def test_config_file(self, mock_post, mock_get):
+        """Asserts that configuration files are correctly handled:
+            - Missing configuration files cause an exception to be raised.
+            - Configuration files without all the necessary values cause an exception to be raised.
+            - Valid configuration file allows for successful instantiation of ArchivesSpaceClient class.
+        """
+        with self.assertRaises(ASnakeAuthError) as err:
+            ArchivesSpaceClient()
+        self.assertEqual(str(err.exception), "Failed to authorize ASnake with status: 404")
```

Comment:
Does this pass? Because it shouldn't!

Reply:
Why shouldn't this pass?

Reply:
Good question! Because, if we've patched both `get` and `post` then no actual HTTP requests should be made in the `__init__` method, meaning we wouldn't be getting a 404 not found error.